audit-info/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "audit-info"
```

Comment:
if we are to change to `auditable-info` in case or another have to change here as well